### DISCUSSION TITLE
Number of replicate samples per treatment

### DISCUSSION THREAD

**Discussion by RTignat** [FrederickHuangLin/ANCOMBC#173] [Q&A]
Hi,
I was wondering if there was a minimum of samples per condition for Ancombc to work on?
I have 4 samples (replicates) per condition and I would like to know which ASVs have a differential abundance between the 2 conditions, but it doesn't seem to work... while it works when I have more than 4 samples per condition. Is there a chance to change the minimum of replicate samples per condition?
Thanks you,
Best

**Comment by FrederickHuangLin** [comment]
Hi @RTignat,

Thank you for your inquiry. 

Based on our simulation benchmark, ANCOM-BC or ANCOM-BC2 maintains decent power while controlling the false discovery rate (FDR) well with a sample size per group as small as 10. However, it is important to note that when the sample size is smaller than 10, the FDR control is not guaranteed. Therefore, it is recommended to exercise caution when interpreting the results from studies with small sample sizes.

Best regards,
Huang

**Comment by liz0f** [comment]
Hi! What are some ANCOM-BC2 alternatives for differential abundance analysis if our sample size is small?
